app/controllers/users_controller.rb

```diff
@@ -2,4 +2,12 @@ class UsersController < ApplicationController
   def show
     @user = User.find(params[:id])
   end
+
```

Comment:
Refactor this to look like https://github.com/scalableinternetservices/Grape-Froo/blob/master/app/controllers/templates_controller.rb#L74 so the "User.find" stuff doesn't need to be duplicated